cmd/serve_proxy.go

```diff
@@ -233,16 +242,28 @@ OTHER CONTROLS
 		}
 
 		var cert tls.Certificate
+		var tlsEnabled bool
 		tlsCert := viper.GetString("HTTP_TLS_CERT")
 		tlsKey := viper.GetString("HTTP_TLS_KEY")
-		if tlsCert != "" && tlsKey != "" {
-			if tlsCert, err := base64.StdEncoding.DecodeString(tlsCert); err != nil {
-				logger.WithError(err).Fatalln("Unable to base64 decode the TLS Certificate")
-			} else if tlsKey, err := base64.StdEncoding.DecodeString(tlsKey); err != nil {
-				logger.WithError(err).Fatalln("Unable to base64 decode the TLS Private Key")
-			} else if cert, err = tls.X509KeyPair(tlsCert, tlsKey); err != nil {
+		if tlsCert == "" && tlsKey == "" {
+			// fallback on file opts
+			certPath := viper.GetString("HTTP_TLS_CERT_PATH")
+			keyPath := viper.GetString("HTTP_TLS_KEY_PATH")
+			if cert, err = tls.LoadX509KeyPair(certPath, keyPath); err != nil {
+				logger.WithError(err).Fatalln("Could not load x509 key pair files")
```

Comment:
if all four variables are empty, this will always fatal, no?

Reply:
yep. Let me fix this...